source/FreeRTOS_DNS_Cache.c

```diff
@@ -207,44 +209,47 @@
                                      ( unsigned ) FreeRTOS_ntohl( ulTTL ) ) );
         }
 
-        if( x == -1 )
+        if( xResult == -1 )
         {
-            x = 0;
+            xResult = 0;
         }
         else
         {
-            x = 1;
+            xResult = 1;
         }
 
-        return x;
+        return xResult;
     }
 
 /**
  * @brief returns the index of the hostname entry in the dns cache.
  * @param[in] pcName find it in the cache
- * @returns index number or -1 if not found
+ * @param [out] xResult index number
+ * @returns res 0 if index in found else -1
```

Comment:
Why not make it consistent with all the other functions and return a pdTRUE/pdFALSE value instead of 0 and -1?